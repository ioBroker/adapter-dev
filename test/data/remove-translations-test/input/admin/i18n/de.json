{
    "goodbye": "Auf Wiedersehen",
    "hello": "Hallo Welt",
    "testKey": "Dieser Schl√ºssel wird entfernt"
}