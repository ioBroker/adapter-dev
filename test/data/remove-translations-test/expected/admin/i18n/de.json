{
    "goodbye": "Auf Wiedersehen",
    "hello": "Hallo Welt"
}